test/core/xds/xds_client_ads_stream_wait_test.cc

```diff
@@ -59,21 +58,37 @@ TEST_F(XdsClientNotifyWatchersDone, Basic) {
           .set_nonce("A")
           .AddFooResource(XdsFooResource("foo1", 6))
           .Serialize());
+  stream->SendMessageToClient(
```

Comment:
Please add a comment indicating that this message won't actually get read by the `XdsClient` until we reset the handle from the watcher notifications.

Reply:
Done